Migration-Project-Guides/GoogleToGoogle.md

```diff
@@ -83,7 +90,7 @@ You should ensure all of your Drive users have Drive enabled in the destination.
 
 Usage of the destination drive during a migration is not recommended. Items may not be moved into place for all users until the full migration for all users has completed. If items are moved around in the destination domain during a migration multiple problems can occur with migration and may cause migration times to increase significantly.
 
-If you are renaming users as part of the migration, you must make sure all import and export names are present and have been updated to the names in the new system.  You should ensure you have provisioned all of your users before migrating your groups. You should not rename groups during a migration. This must be done before the migration begins. If you are migrating Drive items, the tool will warn you before the migration.
+If you are renaming users as part of the migration, you must make sure all import and export names are present and have been updated to the names in the new system.  You should ensure you have provisioned all of your users before migrating your groups and do not rename groups during a migration. Renaming Groups must be done before the migration begins. If you are migrating Drive items, the tool will warn you before the migration.
```

Comment:
How does the tool warn you before migration ? this part feels very odd  we are talking about users and removing  them to if you are migrating drive items the tool will warn you.   very confusing.

Reply:
Agreed - will clarify.

Reply:
I like it but how does the tool warn you before you click start?  do you have to do a discovery run?  the tool is not dynamic so there must be a way it is triggered and the warning is presented to you and it says it warns me before but how i dont understand.